### DISCUSSION TITLE
Dolphin / Ftp connection at Places

### DISCUSSION THREAD

**Discussion by imhatep** [prasanthrangan/hyprdots#518] [Q&A]
Hello, have a question, how to add on the Places/Remote my ftp connections? 
After connecting, this connection does not appear anywhere

**Comment by T-Crypt** [comment]
@imhatep you can add an FTP connection in the folder 'Remote' on the right side of Dophlin.

![231117_10h04m49s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/132922589/752893cd-46f7-4dc2-ae10-b14b767198a2)

![231117_10h06m47s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/132922589/db4f0ba4-4f67-449d-9345-269fce4cc2e4)

**Comment by imhatep** [comment]
Not really what i want
![image](https://github.com/prasanthrangan/hyprdots/assets/35999654/17fd876b-e292-4856-8b6b-f75ea4d35d5a)

I'm not sure that it exists in dolphin at all, I used thunar for a long time and there is something like this, after connecting to ftp 
![image](https://github.com/prasanthrangan/hyprdots/assets/35999654/06d21fc2-1a24-4cc8-98a3-4c4c555c897d)
It is added to places automatically, and there is an option to disconnect(eject icon), but I really need this option to disconnect, because there are different users on the same IP

Is it even possible to disconnect from the ftp connection via dolphin?

**Comment by T-Crypt** [comment]
I would check this out or just download thunar for this! 

`Yay -S thunar`

https://linuxconfig.org/mount-remote-ftp-directory-host-locally-into-linux-filesystem

**Comment by T-Crypt** [comment]
I believe Dolphin only connects to the FTP share when you are actively inside the FTP share. So when you close Dolphin, it is essentially closing the FTP connection -- it is not a keep alive connection state for FTP.

**Comment by imhatep** [comment]
I will temporarily use thunar, maybe later I will deal with dolphin
Maybe someone will be interested for thunar need additional to install gvfs & sshfs
![image](https://github.com/prasanthrangan/hyprdots/assets/35999654/e96c6b90-f41b-470c-87d6-eb9aea7cbf1c)

@T-Crypt 
it opens the last place and saves the connection, somewhere there is a cache that stores information about the connection, as an option to clean it when closing, there is another question, there is an option to hide the top bar using the example of the same thunar?
![image](https://github.com/prasanthrangan/hyprdots/assets/35999654/42045934-f7f6-4db6-9d92-3f5ecb70bbfe)

**Comment by T-Crypt** [comment]
You can hide the menu bar in thunar by going to View > Deselect Menu Bar!

**Comment by prasanthrangan** [comment]
Hi, I'll move this topic to discussions page!
